mesheryctl/internal/cli/root/system/context/delete.go

```diff
@@ -0,0 +1,32 @@
+package context
+
+import (
+	"errors"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// deleteContextCmd represents the update command
+var deleteContextCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "delete context",
+	Long:  `Delete a specific context from mesheryctl config`,
+	Args:  cobra.MinimumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		log.Printf("Delete Context %s from config", args[0])
+		_, exists := configuration.Contexts[args[0]]
+		if !exists {
+			return errors.New("No context to delete.")
+		}
+		delete(configuration.Contexts, args[0])
+		viper.Set("contexts", configuration.Contexts)
+		err := viper.WriteConfig()
+		return err
+	},
+}
+
+func init() {
+
+}
```

Comment:
Why a blank `init()` function is used here?
I am new to Go and asking this out of curiosity.